src/gcp/cloudbuild.ts

```diff
@@ -158,9 +158,15 @@ export async function fetchLinkableRepositories(
   projectId: string,
   location: string,
   connectionId: string,
+  pageToken = "",
 ): Promise<LinkableRepositories> {
   const name = `projects/${projectId}/locations/${location}/connections/${connectionId}:fetchLinkableRepositories`;
-  const res = await client.get<LinkableRepositories>(name);
+  const res = await client.get<LinkableRepositories>(name, {
+    queryParams: {
+      pageSize: 20,
```

Comment:
I would consider passing this in by arg as well. We may as well request a page size of 1000 if we're trying to fetch all repos